tests/integration/ceph_multi_cluster_test.go

```diff
@@ -198,8 +198,9 @@ func (o MCTestOperations) Teardown() {
 
 func (o MCTestOperations) startCluster(namespace, store string) error {
 	logger.Infof("starting cluster %s", namespace)
+	multipleManagers := true
 	err := o.installer.CreateRookCluster(namespace, o.systemNamespace, store, o.testOverPVC, o.storageClassName,
-		cephv1.MonSpec{Count: 1, AllowMultiplePerNode: true}, true, false, false, installer.NautilusVersion())
+		cephv1.MonSpec{Count: 1, AllowMultiplePerNode: true}, true, multipleManagers, false, false, installer.NautilusVersion())
```

Comment:
So we are just testing that 2 mgrs get deployed?

Reply:
The multi-cluster test will deploy two mgrs and all the other test suites will deploy a single mgr